proto/frequenz/api/common/v1/microgrid/components/components.proto

```diff
@@ -141,8 +142,27 @@ message Component {
 
   // The operational lifetime of the component.
   frequenz.api.common.v1.microgrid.Lifetime operational_lifetime = 9;
+
+  // List of rated bounds present for the component identified by Metric.
+  repeated MetricRatedBounds rated_bounds = 10;
+}
+
+// Defines rated bound for a specific metric.
+message MetricRatedBounds {
+  // Metric type the rated bounds are for
+  frequenz.api.common.v1.metrics.Metric metric = 1;
+
+  // The rated bounds of the specified metric.
+  //
+  // This contains the lower and upper bounds
+  // for said metric as specified by the component
+  // manufacturer's ratings.
```

Comment:
I'd suggest going for 80 character widths here. So, if a word makes the line cross the 80 character mark, then we make a new line from that word onwards. It would make the comments look more consistent.

Reply:
Sounds good, done